website/views.py

```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/env python
+
+
+from flask import Blueprint, render_template, session, request, redirect, url_for
+from . import interfacer, loginManager, userManager, tagManager, gameManager
+from . import constants as const
+views = Blueprint('views',__name__)
+
+
+@views.route('/')
+def home():
+    return render_template("home.html")
+
+
+@views.route('/users', methods=['GET', 'POST'])
+def user():
+    if request.method == "POST":
+        viewerID = int(request.form.get('viewerID',''))
+        if(session['userID'] == viewerID):
+            friendID = int(request.form.get('friendID',''))
+            match request.form.get('subType'):
+                case 'removeFriend':
+                    success = userManager.deleteFriend(viewerID,friendID)
+                    if(success):
+                        print("Friend removed.")
+                    else:
+                        print("Failed to remove friend.")
+                case 'addFriend':
+                    success = userManager.addFriend(viewerID,friendID)
+                    if(success):
+                        print("Friend added.")
+                    else:
+                        print("Failed to add friend.")
+
+        else:
+            print("Session and viewerID mismatch")
+
+
+
+
+    userList = userManager.getAllUsers()
+    if session['userID']:
+        userFriends = userManager.getFriendsOf(session['userID'])
+        friendIDs = userManager.getFriendIDs(userFriends)
+        return render_template("users.html", users=userList, friendIDs = friendIDs)
+    return render_template("users.html", users=userList)
+
+@views.route('/games')
+def games():
+    gameList = gameManager.getAllGames()
+    if(session['userID']):
+        # A user is signed in. We can check their library. 
+        userLibrary = userManager.getLibrary(session['userID'])
+        libraryIDs = {game[1] for game in userLibrary}
```

Comment:
Shouldn't this be `game[consts.SQL_ID]` (which is 0, so therefore this is semantically incorrect?)